sklearn/metrics/ranking.py

```diff
@@ -202,6 +202,9 @@ def average_precision_score(y_true, y_score, average="macro",
     >>> average_precision_score(y_true, y_scores)  # doctest: +ELLIPSIS
     0.83...
 
+    .. versionchanged:: 0.19
```

Comment:
Perhaps this needs to be higher up. See where it appears at https://19217-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.metrics.average_precision_score.html... no one will see it. Or maybe it just needs to be under a Notes heading

Reply:
Ah, I was following http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.logistic_regression_path.html#sklearn.linear_model.logistic_regression_path, which also put the `versionchanged` at the very end. Will add a `Notes` heading. (Neat that the rendered documentation for the PR can be looked at online.)